### PR TITLE
Battle Factory: Update RU and LC

### PR DESCRIPTION
**Pull request by KrisXV**
RU QC gave me updated sets for Battle Factory because the last time RU Battle Factory sets were updated was when they were added.

Wingull was removed because it's LC Uber.

### CODE REVIEW

**Comment by Zarel** [comment]
@TheImmortal? Are you the right reviewer for this?

File: data/factory-sets.json [hunk 1]
```diff
@@ -4770,233 +4738,137 @@
...
+		"araquanid": {
 			"flags": {},
 			"sets": [{
-				"species": "Doublade",
-				"item": ["Eviolite"],
-				"ability": ["No Guard"],
-				"evs": {"hp": 252, "atk": 140, "def": 20, "spd": 96},
+				"species": "Araquanid",
+				"item": ["Splash Plate"],
+				"ability": ["Water bubble"],
```
Comment (TheImmortal) [inline data/factory-sets.json]: Bubble

File: data/factory-sets.json [hunk 2]
```diff
@@ -5007,681 +4879,603 @@
...
-				"evs": {"spa": 252, "spd": 4, "spe": 252},
-				"nature": "Timid",
-				"moves": [["Psyshock"], ["Moonblast"], ["Focus Blast"], ["Trick"]]
-			}, {
-				"species": "Gardevoir",
-				"item": ["Choice Specs"],
-				"ability": ["Trace"],
+				"species": "Houndoom",
+				"item": ["Darkinium Z"],
+				"ability": ["FLash Fire"],
```
Comment (TheImmortal) [inline data/factory-sets.json]: Flash

**Comment by TheImmortal** [comment]
GP 1/2. Maybe @Marty-D wants to take a quick look.

**Comment by Marty-D** [comment]
Remove Stakataka from RU, it was banned 3 days ago but no one reported that fact anywhere relevant.

File: data/factory-sets.json [hunk 3]
```diff
@@ -5007,681 +4879,603 @@
...
+		"florges": {
 			"flags": {},
 			"sets": [{
-				"species": "Roserade",
-				"item": ["Life Orb"],
-				"ability": ["Technician"],
-				"evs": {"def": 4, "spa": 252, "spe": 252},
+				"species": "Florges",
+				"item": ["Leftovers"],
+				"ability": ["FLower Veil"],
```
Comment (Marty-D) [inline data/factory-sets.json]: Flower

**Comment by Zarel** [comment]
Should I just upgrade the spellchecker with capitalization checking? I didn't really expect you guys to be doing this kind of reviewing...

**Comment by TheImmortal** [comment]
> 
> 
> Should I just upgrade the spellchecker with capitalization checking? I didn't really expect you guys to be doing this kind of reviewing...

Sure

**Comment by KrisXV** [comment]
this is good now i think

**Pull request merged**